docs/tutorials/zero-to-hero.md

```diff
@@ -178,127 +70,43 @@ window.walletAccount.requestSignIn(
 );
 ```
 
-### Cleaning up our sample JavaScript
-
-Since we're modifying a sample application, we should take care to avoid errors caused by the edits we made.  One type of error that we've introduced is related to deleted HTML.  Let's comment out the lines of JavaScript that are expecting that HTML to exist.
-
-> In the file `src/main.js`
-> - Comment out the following lines of code.  There are 4 of them in total.
-
-```js
-// HEADS UP!  Only comment out the lines indicated below.  Leave the others as is.
-
-async function doWork() {
-  // document.getElementById('refresh-button').addEventListener('click', updateWhoSaidHi);
-}
-
-function signedInFlow() {
-  // document.getElementById('say-hi').addEventListener('click', () => {
-  //   window.contract.sayHi().then(updateWhoSaidHi);
-  // });
-}
-```
-
-<blockquote class="warning">
-<strong>heads up</strong><br><br>
-
-If you see any JavaScript errors in your console while running the application in later steps, it's likely that you did not comment out the correct lines of code or you commented out too many lines of code.
+### Test the login
 
-Take care to avoid these errors because once an error is raised by the JavaScript interpreter, the browser **stops executing** all other JavaScript so you will not see the expected results.
+Let's try running the app with the changes so far.
 
-</blockquote>
-
-### Calling the functions
-
-Let's try calling these functions!
-
-> In NEAR Studio
-> - Click **Run** to launch the project
+> In Gitpod command line, run `yarn dev` to restart the app
```

Comment:
Since Gitpod automatically runs `yarn dev` they need to know to cancel out of that.
> In Gitpod command line, first cancel the current process with command/control + C, then run `yarn dev` to restart the app.

Reply:
Do we actually need to do that? We don't say this for command line instructions. If we do decide to start calling out this step, I think we need to do it across the whole docs site since we have yarn dev everywhere

Reply:
added ctrl c text